projects/step-frontend/src/lib/modules/timeseries/components/chart-dashlet-settings/chart-dashlet-settings.component.html

```diff
@@ -35,6 +34,19 @@
             />
           </step-form-field>
         </div>
+        <div class="form-row">
+          <step-form-field>
+            <step-label>Secondary Axes Aggregation</step-label>
+            <mat-select
+              [value]="item.chartSettings!.secondaryAxes?.aggregation"
+              (valueChange)="onSecondaryAggregateSelect($event)"
+            >
+              <mat-option [value]="undefined">-</mat-option>
+              <mat-option *ngFor="let attr of AGGREGATE_TYPES" [value]="attr">{{ attr }} </mat-option>
+            </mat-select>
+          </step-form-field>
+          <div></div>
```

Comment:
What is the purpose of this empty `<div></div>`?

Reply:
to force the parent container to split into 2 equal elements